packages/dds/tree/src/feature-libraries/chunked-forest/chunkTree.ts

```diff
@@ -339,7 +345,15 @@ export interface ChunkPolicy {
 	shapeFromSchema(schema: TreeNodeSchemaIdentifier): ShapeInfo;
 }
 
-function newBasicChunkTree(cursor: ITreeCursorSynchronous, policy: ChunkPolicy): BasicChunk {
+export interface ChunkCompressor {
+	policy: ChunkPolicy;
+	idCompressor?: IIdCompressor;
```

Comment:
since most usages probably should provide the idCompressor, we can make it less likley people will forget if we make it required, and we can still allow it to be undefined:
```suggestion
	idCompressor: IIdCompressor | undefined;
```

Reply:
Updated `ChunkCompressor`'s `idCompressor` type to `IIDCompressor | undefined`